src/OrganisationRegistry/BaseCommand.cs

```diff
@@ -18,11 +18,11 @@ Guid IMessage.Id
             set => Id = value;
         }
 
-        public IUser User { get; set; }
+        public IUser? User { get; set; }
```

Comment:
I'm not sure if we're conveying the correct intent here. Functionally, commands should always have a User assigned. However, from a technical perspective this isn't always true, because of legacy reasons. Can we design around this?